src/share/classes/com/sun/javatest/regtest/exec/Agent.java

```diff
@@ -182,12 +187,24 @@ private Agent(File dir, JDK jdk, List<String> vmOpts, Map<String, String> envVar
                 final int ACCEPT_TIMEOUT = (int) (60 * 1000 * timeoutFactor);
                 // default 60 seconds, for server to start and "phone home"
                 ss.setSoTimeout(ACCEPT_TIMEOUT);
-                log("Waiting up to " + ACCEPT_TIMEOUT + " milli seconds for a" +
-                        " socket connection on port " + port +
-                        (pid != UNKNOWN_PID ? " from process " + pid : ""));
-                Socket s = ss.accept();
-                log("Received connection on port " + port + " from " + s);
-                s.setSoTimeout((int)(KeepAlive.READ_TIMEOUT * timeoutFactor));
+                final int readTimeout = (int) (KeepAlive.READ_TIMEOUT * timeoutFactor);
+                Socket s = null;
+                for (int i = 0; i < maxAcceptAttempts; i++) {
+                    log("Waiting up to " + ACCEPT_TIMEOUT + " milli seconds for a" +
```

Comment:
Typo: milliseconds.

Reply:
Oh, nevermind, it's not new.

Reply:
Hello Ludvig, I think it was me who introduced that log message in the first place (in some previous integrated code). I've fixed it now and updated the PR.